packages/odyssey/src/scss/base/_typography-text.scss

```diff
@@ -113,6 +119,24 @@ small {
   font-size: $size-body-caption;
 }
 
+details {
+  font-size: ms(0);
+
+  p {
+    font-size: ms(0);
```

Comment:
Font size issue. I don't think we want to start shipping two variants of `details/summary` for 14px vs 16px, so I set the internal paragraphs to `14px` here to ensure the element can be used alongside UI without looking odd.

Since these are disclosures, the text inside could be assumed to be secondary in priority.